README.rst

```diff
@@ -138,3 +138,87 @@ which will each run jobs as specified by the factors given.
   ``py27-django18``, and ``py34-django18``,
   because the ``os`` factor is present,
   and limits it to just those envs.
+
+
+After All
+=========
+
+Inspired by `travis-after-all`_ and `travis_after_all`_,
+this feature allows a job to wait for other jobs to finish
+before it calls itself complete.
+
+.. _`travis-after-all`: https://github.com/alrra/travis-after-all
+.. _`travis_after_all`: https://github.com/dmakhno/travis_after_all
+
+There are three environment variables
+that can be used to configure this feature.
+
+* ``GITHUB_TOKEN``. This is *required*,
+  and should be encrypted in the ``.travis.yml``.
+  This is used as the authentication method
+  for the Travis CI API.
+* ``TRAVIS_POLLING_INTERVAL``.
+  How often, in seconds, we should check the API
+  to see if the rest of the jobs have completed.
+  Defaults to 5.
+* ``TRAVIS_API_URL``.
+  The base URL to the Travis API for this build.
+  This defaults to ``https://api.travis-ci.org``.
+  A common override will be to the commercial version,
+  at ``https://api.travis-ci.com``.
+
+Configure which job to wait on by adding
+the ``[travis:after]`` section to the ``tox.ini`` file.
+The ``travis`` key looks for values that would be keys
+in various items in the ``[travis]`` section,
+and the ``env`` key looks for values that would be keys
+in items in the ``[travis:env]`` section.
+
+For example:
+
+.. code-block:: ini
+
+    [travis:after]
+    travis = python: 3.5
+    env = DJANGO: 1.8
```

Comment:
It might be useful to add a stub of a .travis.yml deploy section that matches these factors. Otherwise, the docs are great.